site/src/api/queries/organizations.ts

```diff
@@ -121,6 +124,60 @@ export const provisionerDaemons = (organization: string) => {
   };
 };
 
+const orgChecks = (
+  organizationId: string,
+): Record<string, AuthorizationCheck> => ({
+  viewMembers: {
+    object: {
+      resource_type: "organization_member",
+      organization_id: organizationId,
+    },
+    action: "read",
+  },
+  editMembers: {
+    object: {
+      resource_type: "organization_member",
+      organization_id: organizationId,
+    },
+    action: "update",
+  },
+  createGroup: {
+    object: {
+      resource_type: "group",
+      organization_id: organizationId,
+    },
+    action: "create",
+  },
+  viewGroups: {
+    object: {
+      resource_type: "group",
+      organization_id: organizationId,
+    },
+    action: "read",
+  },
+  editGroups: {
+    object: {
+      resource_type: "group",
+      organization_id: organizationId,
+    },
+    action: "update",
+  },
+  editOrganization: {
+    object: {
+      resource_type: "organization",
+      organization_id: organizationId,
+    },
+    action: "update",
+  },
+  auditOrganization: {
+    object: {
+      resource_type: "audit_log",
+      organization_id: organizationId,
+    },
+    action: "read",
+  },
```

Comment:
If we move auditing back to it's own page outside this settings page, we can drop this one :+1:

Reply:
Will do!!